scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/GeneratorConfig.scala

```diff
@@ -31,7 +31,8 @@ object GeneratorConfig {
 
   private val toCamelCase: String => String = _.split("_").foldLeft("") {
```

Comment:
nit: we can make this public and it's nicer if we have unit tests verifying the current behavior. but it can be done separately.

Reply:
nit? i am not familiar with the acronym.

Reply:
https://stackoverflow.com/q/27810522/840108

Reply:
Ok i will address before merge

Reply:
@dieselmat No rush at all but are you going to update this PR? If not, I can modify this after merging your PR.

Reply:
Will do after this week. Just swamped with a lot of things